docs/source/guide/configuration.rst

```diff
@@ -421,7 +421,7 @@ in the ``~/.aws/config`` file.
 ``sts_regional_endpoints``
     Sets AWS STS endpoint resolution logic. This configuration can also be set
     using the environment variable ``AWS_STS_REGIONAL_ENDPOINTS``. By default,
-    this configuration option is set to ``legacy``. Valid values are the following:
+    this configuration option is set to ``regional``. Valid values are the following:
```

Comment:
Did we discuss if we should add a note here about the default changing on July 31st 2025 and linking to [the blog post](https://aws.amazon.com/blogs/developer/updating-aws-sdk-defaults-aws-sts-service-endpoint-and-retry-strategy/)?